The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/5941735752/job/16113221430) [(plain)](https://github.com/rust-lang/rust/commit/06567ad7fafcf0a2b380a8908d004a3ca8c88333/checks/16113221430/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:c85c95e3d7251135ab7dc9ce3241c5835cc595a9)
Download action repository 'actions/upload-artifact@v3' (SHA:0b7f8abb1508181956e8e162db84b466c27e18ce)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_d78b8630-097f-48b8-8924-34ca18687923
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
GITHUB_GRAPHQL_URL=https://api.github.com/graphql
GITHUB_HEAD_REF=broken-wait-status
GITHUB_JOB=pr
GITHUB_PATH=/home/runner/work/_temp/_runner_file_commands/add_path_d78b8630-097f-48b8-8924-34ca18687923
GITHUB_REF=refs/pull/115108/merge
GITHUB_REF_NAME=115108/merge
GITHUB_REF_PROTECTED=false
---
Removing intermediate container ce4d12bde1b5
 ---> 233af20e715a
Step 6/10 : COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
 ---> 4cfaef92bb16
Step 7/10 : RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
Collecting binaryornot==0.4.4
  Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
Collecting boolean-py==4.0
  Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
---
Building wheels for collected packages: reuse
  Building wheel for reuse (pyproject.toml): started
  Building wheel for reuse (pyproject.toml): finished with status 'done'
  Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=180117 sha256=2196c9034bf565528bbb1ee6dad4f753eb813f58822363e6b768f09c73e4d4ff
  Stored in directory: /tmp/pip-ephem-wheel-cache-qyminsz6/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
  Attempting uninstall: setuptools
    Found existing installation: setuptools 59.6.0
    Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
---
  Downloading virtualenv-20.24.3-py3-none-any.whl (3.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.0/3.0 MB 19.0 MB/s eta 0:00:00
Collecting filelock<4,>=3.12.2
  Downloading filelock-3.12.2-py3-none-any.whl (10 kB)
Collecting platformdirs<4,>=3.9.1
  Downloading platformdirs-3.10.0-py3-none-any.whl (17 kB)
Collecting distlib<1,>=0.3.7
  Downloading distlib-0.3.7-py2.py3-none-any.whl (468 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 38.5 MB/s eta 0:00:00
Installing collected packages: distlib, platformdirs, filelock, virtualenv
Successfully installed distlib-0.3.7 filelock-3.12.2 platformdirs-3.10.0 virtualenv-20.24.3
Removing intermediate container 69a55bcb8d4a
 ---> 7295320ba688
Step 8/10 : COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
 ---> 69343611ffc8
 ---> 69343611ffc8
Step 9/10 : COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ---> b557404d20fd
Step 10/10 : ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
Removing intermediate container cf423dca7ce3
 ---> 389b46d65c1a
Successfully built 389b46d65c1a
Successfully tagged rust-ci:latest
Successfully tagged rust-ci:latest
##[endgroup]
Built container sha256:389b46d65c1a03234e69a210474243d5f90150e1f3b05366cbd2b2e5451b620d
Uploading finished image to https://ci-caches.rust-lang.org/docker/f76604104d6a6f4aa068dca8679411c51198f6c5562033174b89cf3ceb2f9ecc175231f1257b18e00fa57bb3eed5f53dbcc121e55de0452dda3db18894e27ea0

<botocore.awsrequest.AWSRequest object at 0x7f3e94c01f90>
gzip: stdout: Broken pipe
xargs: docker: terminated by signal 13
[CI_JOB_NAME=mingw-check-tidy]
[CI_JOB_NAME=mingw-check-tidy]
---
DirectMap4k:      169920 kB
DirectMap2M:     7170048 kB
DirectMap1G:    11534336 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
    Finished dev [unoptimized] target(s) in 0.03s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/3e9e5745dfd3ad9a553a4c7d2b3d125dc1473fa6/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-3e9e5745dfd3ad9a553a4c7d2b3d125dc1473fa6-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished release [optimized] target(s) in 26.57s
##[endgroup]
fmt check
##[error]Diff in /checkout/library/std/src/sys/unix/process/process_unsupported/wait_status.rs at line 37:
         // Likewise the macros disregard all the high bits, so are happy to declare
         // out-of-range values to be WIFEXITED, WIFSTOPPED, etc.
         let w = self.wait_status;
-        if (w & 0x7f) == 0 {
-            Some((w & 0xff00) >> 8)
-            None
-        }
-        }
+        if (w & 0x7f) == 0 { Some((w & 0xff00) >> 8) } else { None }
 
 
     pub fn signal(&self) -> Option<i32> {
##[error]Diff in /checkout/library/std/src/sys/unix/process/process_unsupported/wait_status.rs at line 48:
         let signal = self.wait_status & 0x007f;
-        if signal > 0 && signal < 0x7f {
-            Some(signal)
-            None
-        }
-        }
+        if signal > 0 && signal < 0x7f { Some(signal) } else { None }
 
     pub fn core_dumped(&self) -> bool {
     pub fn core_dumped(&self) -> bool {
##[error]Diff in /checkout/library/std/src/sys/unix/process/process_unsupported/wait_status.rs at line 59:
     pub fn stopped_signal(&self) -> Option<i32> {
     pub fn stopped_signal(&self) -> Option<i32> {
         let w = self.wait_status;
-        if (w & 0xff) == 0x7f {
-            Some((w & 0xff00) >> 8)
-            None
-        }
-        }
+        if (w & 0xff) == 0x7f { Some((w & 0xff00) >> 8) } else { None }
 
     pub fn continued(&self) -> bool {
     pub fn continued(&self) -> bool {
##[error]Diff in /checkout/library/std/src/sys/unix/process/process_unsupported/wait_status.rs at line 84:
 #[cfg(all(test, target_os = "linux"))]
 mod compare_with_linux {
     use super::ExitStatus as Emulated;
-    use crate::process::ExitStatus as Real;
     use crate::os::unix::process::ExitStatusExt as _;
+    use crate::process::ExitStatus as Real;
     #[test]
     fn compare() {
     fn compare() {
Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/library/std/src/sys/unix/process/process_vxworks.rs" "/checkout/library/std/src/sys/unix/process/process_unix.rs" "/checkout/library/std/src/sys/unix/process/process_unsupported/wait_status.rs" "/checkout/library/std/src/sys/unix/process/process_unsupported.rs" "/checkout/library/std/src/sys/unix/process/mod.rs" "/checkout/library/std/src/sys/unix/args.rs" "/checkout/library/std/src/sys/windows/args.rs" "/checkout/library/std/src/sys/unix/process/process_fuchsia.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Tue Aug 22 16:55:13 UTC 2023
  network time: Tue, 22 Aug 2023 16:55:13 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/std/src/sys/unix/process/process_common/tests.rs
+++ b/library/std/src/sys/unix/process/process_common/tests.rs
@@ -175,7 +175,7 @@ fn unix_exit_statuses() {
     use crate::process::*;
 
     for exit_code in 0..=0xff {
-        // TODO impl From<ExitCode> for ExitStatus and then test that here too;
+        // FIXME impl From<ExitCode> for ExitStatus and then test that here too;
         // the two ExitStatus values should be the same
         let raw_wait_status = exit_code << 8;
         let exit_status = ExitStatus::from_raw(raw_wait_status);

--- a/library/std/src/sys/unix/process/process_unsupported/wait_status.rs
+++ b/library/std/src/sys/unix/process/process_unsupported/wait_status.rs
@@ -37,20 +37,12 @@ impl ExitStatus {
         // Likewise the macros disregard all the high bits, so are happy to declare
         // out-of-range values to be WIFEXITED, WIFSTOPPED, etc.
         let w = self.wait_status;
-        if (w & 0x7f) == 0 {
-            Some((w & 0xff00) >> 8)
-        } else {
-            None
-        }
+        if (w & 0x7f) == 0 { Some((w & 0xff00) >> 8) } else { None }
     }
 
     pub fn signal(&self) -> Option<i32> {
         let signal = self.wait_status & 0x007f;
-        if signal > 0 && signal < 0x7f {
-            Some(signal)
-        } else {
-            None
-        }
+        if signal > 0 && signal < 0x7f { Some(signal) } else { None }
     }
 
     pub fn core_dumped(&self) -> bool {
@@ -59,11 +51,7 @@ impl ExitStatus {
 
     pub fn stopped_signal(&self) -> Option<i32> {
         let w = self.wait_status;
-        if (w & 0xff) == 0x7f {
-            Some((w & 0xff00) >> 8)
-        } else {
-            None
-        }
+        if (w & 0xff) == 0x7f { Some((w & 0xff00) >> 8) } else { None }
     }
 
     pub fn continued(&self) -> bool {

--- a/library/std/src/sys/unix/process/process_unsupported/wait_status.rs
+++ b/library/std/src/sys/unix/process/process_unsupported/wait_status.rs
@@ -84,8 +84,8 @@ impl ExitStatus {
 #[cfg(all(test, target_os = "linux"))]
 mod compare_with_linux {
     use super::ExitStatus as Emulated;
-    use crate::process::ExitStatus as Real;
     use crate::os::unix::process::ExitStatusExt as _;
+    use crate::process::ExitStatus as Real;
 
     #[test]
     fn compare() {

